app/services/redis.py

```
@@ -0,0 +1,55 @@
+    """Get Redis client instance."""
+    global redis_client
+    if redis_client is None:
+        redis_client = await aioredis.from_url(
+            settings.REDIS_URL, encoding="utf-8", decode_responses=True
+        )
+    return redis_client
+
+
+async def close_redis():
+    """Close Redis connection."""
+    global redis_client
```

Pylint (3.3.7)

[warning] Using global for 'redis_client' but no assignment is done

(W0602)